store/store.go

```diff
@@ -428,7 +429,17 @@ func (s *SnapUbuntuStoreRepository) Snap(name, channel string, auther Authentica
 
 	s.checkStoreResponse(resp)
 
-	return infoFromRemote(searchData.Payload.Packages[0]), nil
+	pkg := searchData.Payload.Packages[0]
+
+	var purchases []*purchase
+	if auther != nil {
+		purchases, err = s.getPurchases(pkg.SnapID, channel, auther)
```

Comment:
do we really always need this information? I understand why it seems convenient to always ask for it but given how the store services are organized it doesn't seem a good idea